src/til/ut_til/EnumSetTests.cpp

```diff
@@ -53,39 +53,39 @@ class EnumSetTests
 
         Log::Comment(L"Start with no bits set");
         til::enumset<Flags> flags;
-        VERIFY_ARE_EQUAL(0b00000u, flags.to_ulong());
+        VERIFY_ARE_EQUAL(0b00000u, flags.bits());
 
         Log::Comment(L"Set bit 2 to true");
         flags.set(Flags::Two);
-        VERIFY_ARE_EQUAL(0b00100u, flags.to_ulong());
+        VERIFY_ARE_EQUAL(0b00100u, flags.bits());
 
         Log::Comment(L"Flip bit 4 to true");
         flags.flip(Flags::Four);
-        VERIFY_ARE_EQUAL(0b10100u, flags.to_ulong());
+        VERIFY_ARE_EQUAL(0b10100u, flags.bits());
 
         Log::Comment(L"Set bit 0 to true");
         flags.set(Flags::Zero);

```

Comment:
As above, technically this should now be:
```
flags.set(Flags::Zero, true);
```

Reply:
I've changed this one and left the other as it is.